src/Umbraco.Web.UI.Client/src/packages/core/collection/default/collection-default.element.ts

```diff
@@ -69,6 +69,15 @@ export class UmbCollectionDefaultElement extends UmbLitElement {
 		);
 	}
 
+	#generateEmptyStateLabel() {
+		const labelKey =
+			this.#collectionContext?.manifest?.meta.noItemsLabel ??
+			this.#collectionContext?.getConfig()?.noItemsLabel ??
+			'#collection_noItemsTitle';
+
+		return labelKey.startsWith('#') ? this.localize.string(labelKey) : labelKey;
```

Comment:
Also this is not how we generally treat localizations.. it should be possible to do this... `bla bla #translateMe`. it might be a weird case, but that's how we gone about it so far :-)